student/src/main/java/tds/student/performance/domain/TestSessionTimeLimitConfiguration.java

```diff
@@ -64,7 +64,7 @@ public void setOpportunityExpiration(Integer opportunityExpire) {
      *     This value is in minutes.
      * </p>
      */
-    public Integer getOpportunityRestartWindowMinutes() {
+    public Integer getOpportunityRestartMinutes() {
```

Comment:
This getter was mis-named, which caused serialization/deserialization issues when round-tripping through the redis cache.  I double-checked all the other classes that are being cached for similar issues.